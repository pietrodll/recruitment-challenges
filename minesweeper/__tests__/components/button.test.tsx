import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import Button from '../../src/components/button';

describe('Button component', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Button
          title="Test"
          onPress={() => {}}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
