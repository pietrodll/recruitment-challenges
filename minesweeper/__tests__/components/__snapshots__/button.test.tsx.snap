// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component renders correctly 1`] = `
<View
  accessible={true}
  focusable={true}
  isTVSelectable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#2244ee",
      "borderRadius": 15,
      "display": "flex",
      "height": 60,
      "justifyContent": "center",
      "opacity": 1,
      "width": 180,
    }
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#ffffff",
          "fontSize": 30,
          "fontWeight": "bold",
        },
        undefined,
      ]
    }
  >
    Test
  </Text>
</View>
`;
